spynnaker/pyNN/spynnaker.cfg

```diff
@@ -9,6 +9,7 @@ read_bif_field_iobuf = True
 generate_bit_field_report = False
 generate_bit_field_summary_report = False
 write_router_compressor_with_bitfield_iobuf = True
+write_rerun_expander_iobuf = False
```

Comment:
Seems odd to only control the IOBUF extraction of this part without also having IOBUF in the original synapse expander also use this.  This would then give a simpler variable name.

Reply:
you might want to only have the iobuf of the rerun. but if you want to accum them. itll have to happen when the first pr is merged. coz eitehr can do it.